migrations/sqls/20230921102640-delete-broken-charge-versions-up.sql

```diff
@@ -0,0 +1,9 @@
+/* deletes charge versions in the workflow that have incorrect data */
+CREATE TEMP TABLE tempChargeVersionId
+AS (
+    SELECT charge_version_workflow_id AS id
+    FROM water.charge_version_workflows
+    WHERE charge_version_workflow_id = '4fe385f1-a50c-4608-91ee-96a3b15ad3b6'
+);
+DELETE FROM water.charge_version_workflows WHERE charge_version_workflow_id = '4fe385f1-a50c-4608-91ee-96a3b15ad3b6';
+DROP TABLE tempChargeVersionId;
```

Comment:
The query this was copied from relied on selecting IDs based on a WHERE filter. We know the ID in this case so we can drop a lot of what was there.

```suggestion
DELETE FROM water.charge_version_workflows WHERE charge_version_workflow_id = '4fe385f1-a50c-4608-91ee-96a3b15ad3b6';

```

Reply:
I thought it would be that simple but I assumed they created a temp table to keep the IDs in case it failed 😬